lib/actions/term-groups.js

```diff
@@ -12,14 +12,16 @@ import getRootGroups from '../selectors';
 import {setActiveSession, ptyExitSession, userExitSession} from './sessions';
 
 function requestSplit(direction) {
-  return () => (dispatch, getState) => {
+  return (sessionUid, sourceUid) => (dispatch, getState) => {
```

Comment:
I think that `sourceUid` is indeed a more accurate name than `activeUid`. But maybe, it should have the same name as here: `export function addSession({uid, shell, pid, cols, rows, splitDirection, termGroupUid, activeUid}) {`

It would be better to change `activeUid` to `sourceUid` but you're right to keep this name to have backward compatibility.

Reply:
Yeah that makes sense.  I'll change activeUid to sourceUid to keep things consistent.

Reply:
Be careful, we can't rename `activeUid` in redux actions.
Maybe better to change sourceUid to activeUid.

Reply:
Oh, because people may be intercepting activeUid... got it.  Ok, I'll change sourceUid